### PR TITLE
[skipci] Rewrite README.md

### PR DESCRIPTION
**Pull request by fansehep**
### What problem does this PR solve?

Issue Number: #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:
 here [new README.md](https://github.com/fansehep/curve/blob/opm_doc/README.md) .

How it Works:
1. add more icons.
2. add more links.
3. add a contest with other fs. but it's not comprehensive. 

By the way, I am finishing it with @zhanghuidinah. Thanks for her help form the bottom of my heart.

If you have some suggestions or requirements to me, welcome to you comment or review the new README.md in this pull request.

**It is not yet completed!** If you can provide some help or some documents to me, I will be happy to accept your help.

### CODE REVIEW

**Comment by YunhuiChen** [comment]
recheck

**Comment by zhanghuidinah** [comment]
Awesome work ! @fansehep Maybe we can split the pr

**Comment by zhanghuidinah** [comment]
Awesome work ! @fansehep 

Maybe we can split this PR cause our architecture diagram has not been completed yet.

**Review by zhanghuidinah — APPROVED** [review]
/lgtm

File: README.md [hunk 1]
```diff
@@ -1,74 +1,137 @@
...
+  - [Contribute us](#-contribute-us)
+  - [LICENSE](#-license)
+  - [Release Cycle](#-release-cycle)
+  - [Branch](#-branch)
+  - [Feedback & Contact](#-feedback--contact)
+
+## 🎉 About Curve
+
+**Curve** is a mordern storage system developed by netease, currently supporting file storage(CurveFs) and block stroage(CurveBs). Now it's hosted at [CNCF](https://www.cncf.io/) as a sandbox project.
+
```
Comment (ilixiaocui) [inline README.md:49]: storage(CurveFS) and block stroage(CurveBS)

File: README.md [hunk 2]
```diff
@@ -1,74 +1,137 @@
...
+<details>
+  <summary><b><font=5>High Performance | Easy Operation | Cloud Native</b></font></summary>
+
+- High Performance
+  - Please see [Curve vs other fs](#⚔️-curve-contrast).
```
Comment (ilixiaocui) [inline README.md:54]: 这个链接点进去还是本页。不是性能对比图。另外性能对比图为什么不直接放在首页，要以链接的方式展现呢

File: README.md [hunk 3]
```diff
@@ -1,74 +1,137 @@
...
+- Curve can also be used as cloud storage middleware using S3-compatible object storage as the data storage engine, providing cost-effective shared file storage for public cloud users.
+</details>
+
+## ⚙️ [Curve Architecture](https://github.com/opencurve/curve/wiki)
+
+<div align=center> <image src="docs/images/Curve-arch.png" width=60%>
+<div align=left>
 
-<image src="docs/images/Curve-deploy-on-public-cloud.png" width=55%>
+## 🚀 [Curve Deployment](https://github.com/opencurve/curveadm/wiki)
```
Comment (ilixiaocui) [inline README.md:110]: 这些大标题的前置图标也可以不要，有下划线区分了。

Comment (ilixiaocui) [reply README.md:110]: Curve Deployment 这个点进来以为是部署

**Pull request merged**